.ci/windows-steps.yaml

```diff
@@ -133,6 +133,7 @@ steps:
 
 # Run tests via ctest.
 - bash: |
+    dir
```

Comment:
Ahh, I wanted to see the content in `cd build`.

Reply:
I added that too.

Reply:
btw I think you can edit my files in this PR for quick checks, you won't have to wait.